dragging.js

```diff
@@ -65,6 +65,7 @@ const mouseUpHandler = function () {
     document.removeEventListener('mouseup', mouseUpHandler);
 };
 
+/*
```

Comment:
are we maintaining these comments for a little while longer? or could we do away with them now?
If we decide to keep them, can we add a very attention  getting comment to the top of this comment block explaining why it's here?

Reply:
I was intending to leave it just in case we changed our minds about using it but I guess when we get around to a framework overhaul we won't need this code anyway. It's easy enough to rewrite anyway. There are a few other comment blocks throughout the code we could probably remove in addition to this one if we want, up to you guys.